src/main/java/lotto/domain/LottoGame.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
```

Comment:
사용하지 않는 import문은 제거해주세요! 🙏 
```suggestion
import java.util.HashMap;
```